JSGames/js/pitcher.js

```diff
@@ -0,0 +1,61 @@
+let fallingSpeed;
+let speed;
+let position = 1;
+let angle;
+let throwingDistance;
+let distanceToCatcher;
+
+
+function setCatcherDistance() {
+    distanceToCatcher = Math.random() * 90 + 10;
+    distanceToCatcher = Math.round(distanceToCatcher);
+    catcherPosition.innerHTML = `Your catcher is ${distanceToCatcher}meter away from you`
+}
+
+// function setThrowingDistance() {
+//     speed = document.getElementById('speedValue').value;
+//     angle = document.getElementById('angleValue').value;
+//     throwingDistance = Math.round(((speed * speed) * Math.sin(2 * angle)) / fallingSpeed);
+// }
+
+function start() {
+    // setThrowingDistance();
+    speed = document.getElementById('speedValue').value;
+    angle = document.getElementById('angleValue').value * (Math.PI / 180);
+    throwingDistance = Math.round(((speed * speed) * Math.sin(2 * angle)) / fallingSpeed);
+
+    let differenz = Math.abs(throwingDistance - distanceToCatcher);
+
+    if (differenz < 10) {
+        setTimeout(function () {
+            createPopUp("Your throw was perfect!", true);
+        }, 1000)

```

Comment:
Das Popup wird zu der id "flex" hinzugefügt. Diese gibt es im pitcher.html aber nicht.
Deswegen wird das PopUp nicht erzeugt und du bekommst Fehler in der Console.